### PR TITLE
Add Next.js quickstart example.

### PR DESCRIPTION
**Pull request by leerob**

### CODE REVIEW

File: nextjs/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (phoenixy1) [inline nextjs/LICENSE:1]: Just as a note to myself, we'll also need to update the README to mention this.

"This repo contains two folders: vanilla_js/ and react/. Each folder contains a minimal app that implements Plaid. The app in the vanilla_js/ folder uses a vanilla JS front end and a Node/Express backend, while the app in the react/ folder uses a React front end an a Node/Express backend. Both apps implement Plaid Link, the Balance product, and OAuth."

File: nextjs/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#### Set up your environment
+
+Install [Node.js v16+](https://nodejs.dev/learn/how-to-install-nodejs).
+
+#### Install dependencies
+
+Ensure you're in the **nextjs/** folder, then install the necessary dependencies:
+
+```bash
+pnpm install
```
Comment (phoenixy1) [inline nextjs/README.md:20]: maybe should tell people to install `pnpm` as a pre-req? (or change the commands so they use npm instead?) I ended up having to install it.

**Review by phoenixy1 — APPROVED** [review]
OMG thank you, what an unexpected treat! I have basically one comment/question (around usage of npm vs. pnpm) but nothing blocking.

BTW you'd like a Plaid swag box, feel free to reach out on Twitter to @plaiddev with your address and t-shirt size.

File: nextjs/README.md (hunk 2 above)
Comment (leerob) [reply nextjs/README.md:20]: Yeah agreed - could also switch to nom/yarn if you prefer 😄

**Comment by leerob** [comment]
Happy to help! 😄

**Pull request merged**